examples/00-load/create-polyhedron.py

```diff
@@ -0,0 +1,170 @@
+
+
+###############################################################################
+# Connectivity arrays
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# We set which points each of the cells needs. The first element in each array
+# is the number of elements the cell will have. E.g., quad array is composed of
+# 0, 1, 2 and 3 points, so it has 4 elements. This is needed because the connectivity
+# array that will contain all cells needs to have only one dimension, so to be able to
+# know which points belong to which cell, we set that the following N positions in the array
+# belong to a single cell, the next N positions belong to another cell, etc.
+
+quad = np.array([4, 0, 1, 2, 3])
+polygon = np.array([5, 4, 5, 6, 7, 8])
+hexa = np.array([8, 9, 10, 11, 12, 13, 14, 15, 16])
+
+###############################################################################
+# Polyhedron connectivity array
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Polyhedrons connectivity array is defined differently from the rest of the cell types.
+# For polyhedrons, we need to set the faces with the following format:
+#
+# `[NElements, NFaces, Face1NPoints, Face1Point1, Face1Point2..., Face1PointN, Face2NPoints,...]`
+#
+# - `NElements` refers to the total number of elements in the array needed to describe the polyhedron.
+# - `NFaces` is the number of faces the polyhedron will have.
+# - `Face1Npoints` is the number of points the first face will have
+# - `Face1Point1..Face1PointN`` are each of the points that describe face1
+#
+# In `polyhedron_connectivity`, the first element is `NFaces`. `NElements` is added to `polyhedron`.
+
+polyhedron_connectivity = [
+    7,
+    5,
+    17,
+    18,
+    19,
+    20,
+    21,
+    4,
+    17,
+    18,
+    23,
+    22,
+    4,
+    17,
+    21,
+    26,
+    22,
+    4,
+    21,
+    26,
+    25,
+    20,
+    4,
+    20,
+    25,
+    24,
+    19,
+    4,
+    19,
+    24,
+    23,
+    18,
+    5,
+    22,
+    23,
+    24,
+    25,
+    26,
+]
+polyhedron = np.array([len(polyhedron_connectivity)] + polyhedron_connectivity)
+
+
+###############################################################################
+# Cells array
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Now we build the input cells array for `UnstructuredGrid`. We join all cells in
+# a one dimensional array. Internally, the `NElements` previously described is used
+# to know which nodes belong to which cells.
+
+cells = np.hstack((quad, polygon, hexa, polyhedron))
+
+
+###############################################################################
+# Cell types
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# We need to determine the cell types for each of the cells we define in the cells array.
+# The number of elements in this array must coincide with the number of cells in the
+# connectivity array.
+
+cell_type = [pv.CellType.QUAD, pv.CellType.POLYGON, pv.CellType.HEXAHEDRON, pv.CellType.POLYHEDRON]
```

Comment:
I would call this `cell_types`. Or even `celltypes`, just to match the `celltypes` attribute of the constructed `UnstructuredGrid` (we don't have to do the latter, but I think we should use plurals for collections wherever it makes sense).